apis/src/main/kotlin/org/yapp/apis/auth/strategy/signin/GoogleSignInStrategy.kt

```
@@ -0,0 +1,53 @@
+) : SignInStrategy {
+
+    private val log = KotlinLogging.logger {}
+
+    override fun getProviderType(): ProviderType = ProviderType.GOOGLE
+
+    override fun authenticate(credentials: SignInCredentials): UserCreateInfoResponse {
+        return try {
+            val googleCredentials = validateCredentials(credentials)
+            val googleUser = googleApiManager.getUserInfo(googleCredentials.accessToken)
+            createUserInfo(googleUser)
+        } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)